appinventor/components/src/com/google/appinventor/components/runtime/Reasoner.java

```diff
@@ -201,6 +202,19 @@ public void RulesFromRuleset(String ruleset) {
     }
   }
 
+  /**
```

Comment:
I wonder if it makes more sense to override `toString` here rather than implement a new method?

Reply:
I was trying to mimic the same procedural style as with the other "functions" but feel free to adapt. It was more meant for debugging to begin with, really.